pyteal/ast/router.py

```diff
@@ -297,12 +297,11 @@ def to_cond_node(self, use_frame_pt: bool = False) -> CondNode:
 
 class ASTBuilder:
     def __init__(self):
-        self.conditions_n_branches: list[CondNode] = []
         self.methods_with_conds: list[CondWithMethod] = []
-        self._bare_cnbs: list[CondNode] = []
+        self.bare_calls: list[CondNode] = []
```

Comment:
I turned `_bare_cnbs` into `bare_calls` for `ASTBuilder` is not exported, and we should be fine with a not hidden member in a not exported class...